src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,2 +1,24 @@
-DELETE FROM tab_rate_generic_parameters WHERE calculation_method='LinksCommon' AND formula_rate='Induction' AND path_id=372 AND parameter_name='Partition coefficient (water/container)';
-INSERT INTO tab_rate_generic_parameters(calculation_method,formula_rate,path_id,parameter_name,alias) VALUES('LinksCommon','Induction',4910,'Partition coefficient (water/container)','K_water');
+UPDATE tab_calculation_method_rates SET formula='f_int > 0 ? RC * rel_exp_org * f_exp_org_basolatateral * 1 / f_int : 0' WHERE calculation_method='ExpressionParameters' AND formula_rate='InitialConcentrationInterstitial_Transporter';

```

Comment:
@Yuri05 THANK YOU FOR UPDATE CASCADE!! this would have been impossible otherwise